<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HgGraftDialog</class>
 <widget class="QDialog" name="HgGraftDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Copy Changesets</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Revisions</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPlainTextEdit" name="revisionsEdit">
        <property name="toolTip">
         <string>Enter changesets by number, id, range or revset expression one per line</string>
        </property>
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
        <property name="lineWrapMode">
         <enum>QPlainTextEdit::NoWrap</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="userGroup">
     <property name="toolTip">
      <string>Select to give user information</string>
     </property>
     <property name="title">
      <string>User</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="currentUserCheckBox">
        <property name="toolTip">
         <string>Select to use the name of the current user</string>
        </property>
        <property name="text">
         <string>Use current user</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="userEdit">
        <property name="toolTip">
         <string>Enter the user name to be used</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="dateGroup">
     <property name="toolTip">
      <string>Select to give date and time information</string>
     </property>
     <property name="title">
      <string>Date and Time</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="3">
       <widget class="QCheckBox" name="currentDateCheckBox">
        <property name="toolTip">
         <string>Select to use the current date and time</string>
        </property>
        <property name="text">
         <string>Use current date and time</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Date/Time:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateTimeEdit" name="dateTimeEdit">
        <property name="toolTip">
         <string>Enter the date and time to be used</string>
        </property>
        <property name="displayFormat">
         <string notr="true">yyyy-MM-dd hh:mm</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>241</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="logCheckBox">
     <property name="toolTip">
      <string>Select to append graft info to the log message</string>
     </property>
     <property name="text">
      <string>Append Graft &amp;Info</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="dryRunCheckBox">
     <property name="toolTip">
      <string>Select to perform a dry-run of the graft operation</string>
     </property>
     <property name="text">
      <string>Perform Dry-Run</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>revisionsEdit</tabstop>
  <tabstop>userGroup</tabstop>
  <tabstop>currentUserCheckBox</tabstop>
  <tabstop>userEdit</tabstop>
  <tabstop>dateGroup</tabstop>
  <tabstop>currentDateCheckBox</tabstop>
  <tabstop>dateTimeEdit</tabstop>
  <tabstop>logCheckBox</tabstop>
  <tabstop>dryRunCheckBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HgGraftDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HgGraftDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>currentUserCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>52</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>currentUserCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>userEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>currentDateCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>currentDateCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateTimeEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
